defaults/main.yml

```diff
@@ -1,10 +1,8 @@
 ---
 # defaults file for template
-storage_backend: "default"
-pool_layers: ["pool-partitions", "vg"]  # md, luks, vdo under vg
-volume_layers: ["partition", "lv", "fs", "mount"]  # luks under fs
-use_partitions: false
-disklabel_type: gpt
+storage_backend: "blivet"
+use_partitions: null
+disklabel_type: null  # leave unset to allow the role to select an appropriate label type
```

Comment:
Please use the `storage_` prefix consistently for all variables. Ansible has no namespaces!